src/Routes.tsx

```diff
@@ -23,9 +25,16 @@ const { BivariateManagerPage } = lazily(
 const initialUrl = new URL(localStorage.getItem('initialUrl') || '');
 
 export function RoutedApp() {
+  const [urlStore] = useAtom(urlStoreAtom);
   const [{ data, loading }] = useAtom(userResourceAtom);
   const userModel = data && !loading ? data : null;
 
+  useEffect(() => {
+    if (userModel) {
+      metricsInit();
+    }
+  }, [userModel]);
+
```

Comment:
on user change metrics are getting init again but the whole metric pipeline is not running
should we add logic to only init them once?

It's not critical though - as it's okay for us to reload on user change, we just not doing it temporary

Reply:
to be fixed in app init refactor